### DISCUSSION TITLE
Fundamentos do C# : Value Types e Reference Types: Prática

### DISCUSSION THREAD

**Discussion by MoreiraRamires** [balta-io/forum#10] [Tira Dúvidas] [answered]
Olá, tentei replicar o aprendizado da aula  Value Types e Reference Types: Prática, do curso Fundamentos do C# 

Achei interessante que aparentemente Python e  C#  lidam com valores de referencia de formas diferentes. 

Na primeira imagem  trabalhei com C# e obtive resultado igual ao ministrado na aula. 
Na segunda imagem trabalhei com Python e o resultado foi o oposto. 

Agora, o que isso significa na prática?

![image](https://user-images.githubusercontent.com/62817742/190029915-749c27d1-9aef-46cc-b7b1-b2b72db10075.png)
![image](https://user-images.githubusercontent.com/62817742/190029933-e3133d7f-0c7b-4f4a-9968-3910dc29b871.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @MoreiraRamires, os tipos citados no curso são somente referentes ao C#, em outras linguagens eles podem ter comportamentos diferentes.
